src/renderer/modal/newRepository.vue

```diff
@@ -102,13 +101,8 @@ export default {
 			if (!fs.existsSync(repositoryPath)) {
 				fs.mkdirSync(repositoryPath);
 			}
-			this.gitInit(repositoryPath);
 			this.closeModal("NewRepository");
-			this.localRepository(repositoryPath);
-		},
-		async gitInit(path) {
-			await gitInit({ path });
-			this.addRepositoryToDatabase(path);
+			this.addRepositoryToDatabase(repositoryPath);
```

Comment:
```suggestion
			this.addRepositoryToDatabase(repositoryPath);
```

Can you move this before closing the modal?